environment_setup/iac-create-environment-pipeline-arm.yml

```diff
@@ -2,36 +2,35 @@
 trigger:
   branches:
     include:
-    - master
+      - master
   paths:
     include:
-    - environment_setup/arm-templates/*
+      - environment_setup/arm-templates/*
 pr:
   branches:
     include:
-    - master
+      - master
   paths:
     include:
-    - environment_setup/arm-templates/*
+      - environment_setup/arm-templates/*
 
 pool:
-  vmImage: 'ubuntu-latest'
+  vmImage: "ubuntu-latest"
 
 variables:
-- group: devopsforai-aml-vg
-
+  - group: devopsforai-aml-vg
+  - name: WORKSPACE_SKU # https://docs.microsoft.com/en-us/azure/machine-learning/overview-what-is-azure-ml#sku
```

Comment:
You should put this in the `devopsforai-aml-vg` variable group template which is already included to make it cleaner / all variables in one file.

Reply:
agree that it shouldn't be in the variable group, but don't think it should be in the template either.  The SKU is not use anywhere except in the IAC pipeline which is only rarely run.  

I have put it as a variable in the pipeline.